content/resources/chapter-1/transacting-2.tsx

```diff
@@ -20,28 +20,29 @@ export default function TransactingTwoResources({ lang }) {
       readingResources={
         <>
           <Text className="mt-[25px] text-xl font-bold">
-            {t('chapter_one.resources.transacting.transactions_heading')}
+            {t('chapter_one.resources.transacting_two.transactions_heading')}
           </Text>
           <Text>
-            {t('chapter_one.resources.transacting.transactions_paragraph')}
+            {t('chapter_one.resources.transacting_two.transactions_paragraph')}
           </Text>
           <Text className="mt-[25px] text-xl font-bold">
-            {t('chapter_one.resources.transacting.bitcoin_script_heading')}
+            {t('chapter_one.resources.transacting_two.bitcoin_script_heading')}
           </Text>
           <Text>
-            {t('chapter_one.resources.transacting.bitcoin_script_paragraph')}
+            {t(
+              'chapter_one.resources.transacting_two.bitcoin_script_paragraph'
+            )}
```

Comment:
Did prettier format this differently than the other lines above? Don't need to change in this PR or at all, I'm just curious

Reply:
I think prettier has some threshold for some length of line before it decides it is too long and splits it

Reply:
i guess the addition of `_two` made it just long enough